### DISCUSSION TITLE
[node] Breaking change (potential regression) from @types/node 18.11.14 to 18.11.15

### DISCUSSION THREAD

**Discussion by sbatezat** [DefinitelyTyped/DefinitelyTyped#63647] [Issues with a @types package] [answered]
I've created a 'Subset' type.
With @types/node 18.11.14 (and before) there is no issue with that type, with 18.11.15, it's not recognized as expected.

```
export type Native = Primitive | Date | RegExp | Symbol | Error | BigInt;

export type Subset<Type> = {
    [attr in keyof Type]?: Type[attr] extends Native ? Type[attr] : Subset<Type[attr]>;
};
```

Expected (18.11.14) : When X is {a: string, b: Y }, Subset<X> is {a: string, b: Subset<Y>}
Current (18.11.15) : When X is {a: string, b: Y }, Subset<X> is {a: string, b: Y}

[a comment by an automated account is not shown]

**Comment by sbatezat** [comment, marked as answer]
Seems it was an issue with my IDE - sorry.

**Reply by peterblazejewicz** [reply]
no worries, great you've solved it!
